api3dao/airnode
# Support HTTP gateways with airnode-client

Closes https://github.com/api3dao/airnode/issues/1184

## How to review

Review commit by commit. Read https://github.com/api3dao/api3-docs/pull/935 and comment on the docs PR if there are things that should further clarified (or I simply forgot to document them).

## How to test

The documentation should be enough to have full understanding on how to configure the Airnode client. You can use https://docs.api3.org/airnode/v0.8/grp-providers/tutorial/quick-deploy-container/ or modify one of the Airnode examples. You can then use postman or curl.

## Bit related

While testing this I realized that we are not logging much information for the gateway requests. Created https://github.com/api3dao/airnode/issues/1351 for that.
